docs/developer.rst

```diff
@@ -104,8 +104,7 @@ Then be sure to pass the tests::
 
 When contributing to pycodestyle, please observe our `Code of Conduct`_.
 
-To run the Travis CI tests, you'll have to create a build the pycodestyle
-package using tox::
+To run the tests, use tox::
```

Comment:
The core developers use tox. Not everyone will want to. That's the point of specifying that you're running them the way the CI runs them - if they get incongruous results, they can replicate that.